bitnami/appsmith/Chart.lock

```diff
@@ -0,0 +1,12 @@
+dependencies:
+- name: redis
+  repository: https://charts.bitnami.com/bitnami
+  version: 17.3.11
+- name: mongodb
+  repository: https://charts.bitnami.com/bitnami
+  version: 13.4.4
```

Comment:
Why not use the latest version 13.5.0?

Reply:
Ran `helm dependency update` and now it is correctly synced. When I created the chart that version didn't exist.